### ISSUE TITLE
Bluesky: on DM request, if requestee's DMs aren't open, reply with error

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#1326]
...by checking `chat.bsky.actor.declaration`. Suggested by @qazmlp and @mackuba in https://github.com/snarfed/bridgy-fed/issues/966#issuecomment-2347384747.

**Comment by stefanbohacek** [comment]
Perhaps it would make sense to @ mention the invited account publicly when DMs are closed?

**Comment by snarfed** [comment]
Hmm, maybe. The bots don't currently make any public transactional posts or @-mentions like this, though, and I'm a bit reluctant to start. For this use case specifically, I'd probably just encourage the requester to post the @-mention themselves.

**Comment by stefanbohacek** [comment]
Yes, I've seen a few people do that. Personally it feels a bit awkward for me to reach out to strangers who will likely just ignore me. And I am not sure if it's any less annoying, from the point of the invited person, to get this message from a bot, or someone you don't now.

But fair enough, I definitely understand!
